xtask/src/commands/r_cmd_check.rs

```diff
@@ -0,0 +1,17 @@
+use xshell::{Error, Shell};
+
+pub(crate) fn run(shell: &Shell, no_build_vignettes: bool) -> Result<(), Error> {
+    let mut cmd = shell
```

Comment:
why not use the `cmd!` macro here?

Reply:
I did not want to duplicate the string to avoid mistakes when you fix one branch of if but forget about another. And macro does not accept non-literals